src/main/resources/public/css/styles.css

```diff
@@ -1,7 +1,63 @@
+/* Stylesheet for the application - dont' touch */
 /* Stylesheet for the application */
 
-h1 {
-    color: #369;
-    font-family: Arial, Helvetica, sans-serif;
-    font-size: 3rem;
-}
\ No newline at end of file
+html * {
+    box-sizing: border-box;
+}
+
+body {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    background: #FBE7E9;
+    font-family: "Josefin Sans", sans-serif;
+}
+
+.headerImage {
+    width: 500px;
+    height: auto;
+    margin-top: 3%;
+}
+
+.generic-form {
+    max-width: 400px;
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+}
+
+.generic-form input {
+    background-color: rgba(226, 159, 159, 0.5);
+    border: 0;
+    padding: 10px;
+    padding-left: 10px;
+    padding-right: 10px;
```

Comment:
`padding-left` and `padding-right` are overridden by `padding: 10px` meaning those two can be removed